Tmain/omit-long-patterns-etags.d/stdout-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+input.sh,349
+                                                                                                func96(func961,0
```

Comment:
Yes, sorry. The first version seems to have been the correct one.

I simply missed the truncation because it was by 1 or 2 characters.